### DISCUSSION TITLE
Bluesky?

### DISCUSSION THREAD

**Discussion by M07SRT** [robertodoering/harpy#892] [General]
Bluesky

**Comment by ghost** [comment]
I had the same thought, it shouldn't be too hard to repurpose the app for Bluesky...

**Reply by robertodoering** [reply]
harpy won't be updated to support a different service, that's too much work and not worth it.
That said I've been getting more interested in bluesky recently and started to work on a new bluesky client from scratch with some inspiration from harpy.
I plan to open-source it when it's in a decent spot but there is still a ton of work left of it before it is in a releasable state.

**Reply by ghost** [reply]
I understand, I was in the process of forking harpy and turning it into a Bluesky app. I will be happy to help on the new project when it goes open source!

**Reply by robertodoering** [reply]
Awesome, thanks :)
